### DISCUSSION TITLE
No more update !?

### DISCUSSION THREAD

**Discussion by ghost** [LyleMi/Learn-Web-Hacking#14] [Q&A] [answered]
hi , I noticed that you stopped the weekly updates !?

**Comment by LyleMi** [comment, marked as answer]
Hello, first of all, thank you for your interest in this repo.

Please allow me to explain the habit of updating. Sometimes when I finish writing a relatively complete part, I will push a commit. Sometimes when I make some small changes, such as adding reference links, adding tool lists, and correcting typos, I will update this repo on weekends and comment as "weekly update". So when I feel that the current changes are relatively small and not enough to update a commit, I won’t update on weekends.

Therefore, no submission in the past few weeks does not mean that the update has stopped. If I feel that the current amount of revision is sufficient or there is new content, I will update it.

Also, if you have anything you want to see, or have other ideas, please let me know, thank you very much.

Lyle

**Reply by ghost** [reply]
Thanks for your response Lyle. I appreciate your work. Keep us updated :)

**Reply by LyleMi** [reply]
Yes, I will :P
